java/ql/automodel/src/README.md

```diff
@@ -0,0 +1,183 @@
+# Automodel Java Extraction Queries
+
+This pack contains the automodel extraction queries for Java.
```

Comment:
I think this document is missing a one or two sentence description of why someone would want to run them. Also worth mentioning that these queries are meant to be run as part of automated tooling. Since the tooling is not yet publicly released, we don't need to go into too much detail, though.

Reply:
done! thank you :) cffcc7334dc70d6207a590eb0a4001fa763e2c27